spring-amqp-rabbit/build.gradle

```diff
@@ -8,6 +8,7 @@ plugins {
 dependencies {
 	implementation(platform(org.springframework.boot.gradle.plugin.SpringBootPlugin.BOM_COORDINATES))
 	implementation("org.springframework.boot:spring-boot-starter-amqp")
+	implementation("org.springframework.amqp:spring-rabbit:3.0.0-SNAPSHOT")
```

Comment:
I'm not sure about this. On the one hand, I like the smoke tests providing an accurate view of how things would be if someone consumed Boot snapshots. On the other hand, I can see the value of them showing the latest and greatest, even if Boot hasn't yet moved to snapshots of a particular dependency. If we go for the latter, I think we'd want something to remind us to revert this change once Boot's switched to `spring-rabbit` snapshots and we believe things should work without the version override.

Reply:
Yes; this was intended as a temporary change - the smoke test won't work against the milestone pulled in by boot - do we really want to wait 2 months for Boot's next milestone for a working smoke test?

Reply:
Can't we switch Spring Boot to use the snapshot?

Reply:
> Can't we switch Spring Boot to use the snapshot?

That's tricky because of breaking changes in Micrometer that we're trying to shield ourselves from. AMQP's on a Micrometer milestone at the moment, but the more things we have on snapshots in Boot, the rougher time we're going to have with keeping our build green as Micrometer continues to evolve quite rapidly.

Reply:
Good point. I would then be in favor of changing the version consistently rather than in a dedicated smoke test.